tests/Tests/ORM/Mapping/TypedFieldMapperTest.php

```diff
@@ -36,36 +39,38 @@ private static function chainTypedFieldMapper(): ChainTypedFieldMapper
     /**
      * Data Provider for NamingStrategy#classToTableName
      *
-     * @return array<
+     * @return Generator<
```

Comment:
`Generator` requires 4 template types in its generic signature. I suggest using `iterable` as type instead.